<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KSHRAM_Qt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txtboxLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>561</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtboxInputFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSelectInput">
    <property name="geometry">
     <rect>
      <x>479</x>
      <y>10</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSelectOutput">
    <property name="geometry">
     <rect>
      <x>479</x>
      <y>40</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInputFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Output File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtboxOutputFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRun">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClearLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAbout">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
